README.md

```diff
@@ -22,14 +22,14 @@ Any commands in `localstack/setup-sns.sh` will be run when `localstack` starts,
 
 Running all services locally:
 ```bash
-TMPDIR=/private$TMPDIR docker-compose up 
+TMPDIR=/private$TMPDIR docker-compose up
 ```
 Queues and topics will automatically be created when the `localstack` container starts.
 
 Running all services except this application (hence allowing you to run this in the IDE)
 
 ```bash
-TMPDIR=/private$TMPDIR docker-compose up --scale prisoner-offender-search=0 
+TMPDIR=/private$TMPDIR docker-compose up --scale prisoner-offender-search=0
```

Comment:
Ignore these whitespace changes -  I changed my mind about an edit and for some reason I can't replace it with the original version